TypeCobol.LanguageServer/TypeCobolCustomLanguageServer/TypeCobolCustomLanguageServer.cs

```diff
@@ -52,8 +52,11 @@ protected override InitializeResult OnInitialize(InitializeParams parameters)
         /// <param name="options">Client's Options</param>
         protected override void OnDidChangeConfiguration(string[] options)
         {
-            this.UseOutlineRefresh = options.Contains("-ol");
-            if(options.Contains("-cfg=AsFile"))
+            if (options.Contains("-dol"))
+                this.UseOutlineRefresh = false;
+            else if (options.Contains("-ol"))
+                this.UseOutlineRefresh = true;
```

Comment:
@mayanje 
So what's the default value for 'this.UseOutlineRefresh' ? in case neither '-dol' nor '-ol' is passed. I prefer @smedilol's suggestion to have : 

- without parameter `-ol` : outline is activated by default
- `-ol=true` : activate outline
- `-ol=false` : activate outline

Reply:
The default value is always -ol and in passed when the server is started thru the command line.

Reply:
Ok from what I understand, our plugin does not send the `-ol` option in config change notification so without this code, the outline will be disabled at first config change notiifcation.
We'll have to remove the `-dol` option after all our clients have migrated to the latest version (which will send correctly `-ol` option in notification).